cmd/integrationArtifactTransport.go

```diff
@@ -34,7 +34,7 @@ func runIntegrationArtifactTransport(config *integrationArtifactTransportOptions
 	return CreateIntegrationArtifactTransportRequest(config, apimData)
 }
 
-//CreateIntegrationArtifactTransportRequest - Create a transport request for Integration Package
+// CreateIntegrationArtifactTransportRequest - Create a transport request for Integration Package
```

Comment:
Changes to this file are unrelated. Are they necessary for the format voter?

Reply:
> Changes to this file are unrelated. Are they necessary for the format voter?

Restored file from master